### PR TITLE
Update Babel to 7.x.x

### PR DESCRIPTION
**Pull request by aqrln**
Fixes: https://github.com/metarhia/common/issues/183

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["env"]
+  "presets": ["@babel/preset-env"]
```
Comment (belochub) [inline .babelrc]: You can shorten this to `@babel/env`.

File: .babelrc [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["env"]
+  "presets": ["@babel/env"]
```
Comment (belochub) [inline .babelrc]: Should we also add `"sourceType": script` here similar to https://github.com/metarhia/jstp/pull/378?

Comment (aqrln) [reply .babelrc]: @belochub I don't think we should, since this option is undocumented and it won't change anything: we don't use webpack there, so we don't really care about CJS vs ESM in this case.

**Comment by belochub** [comment]
Landed in https://github.com/metarhia/common/commit/c6a431774b9cb31755f52e44c9be48656dcb596a.
